packages/table/src/table-body/render-helper.ts

```diff
@@ -145,7 +145,7 @@ function useRender<T>(props: Partial<TableBodyProps<T>>) {
        * TIP: One dimensional array is used temporarily to avoid rendering flicker.
        * The case of $index being modified has not been found by testing
```

Comment:
The comment is outdated, can you update the comment here?

Reply:
of course